# ArucoDetected.msg
int32 id
geometry_msgs/Pose pose
string side
